packages/lib/services/survey.ts

```diff
@@ -428,6 +494,9 @@ export async function updateSurvey(updatedSurvey: TSurvey): Promise<TSurvey> {
     };
 
     revalidateTag(getSurveysCacheTag(modifiedSurvey.environmentId));
+    revalidateTag(getSurveysWithAnalyticsCacheTag(modifiedSurvey.environmentId));
```

Comment:
these are also a lot of revalidations that can get quiet inconsistent. Would be great of we can somehow only say:
1. Invalidate the single survey
2. Invalidate all surveys